ankisyncd/sync_app.py

```diff
@@ -63,7 +64,8 @@ def _old_client(cv):
                 version = vs[0]
                 note[name] = int(vs[-1])
 
-        version_int = [int(x) for x in version.split('.')]
+        # convert the version string, ignoring non-numeric suffixes like in beta versions of Anki
+        version_int = [int(re.sub(r'[^\d]+', '', x)) for x in version.split('.')]
```

Comment:
Minor style nit: `0-9` instead of `\d`, it's better to be explicit and many regex implementations (including `re`) match more than `0-9` when `\d` is used. I'd also append `$` to the regexp and `sub` the whole version string before splitting, since only the last part is expected to contain a suffix anyway.